Documentation/Configuration/Yaml/ServicesYaml.rst

```diff
@@ -27,4 +27,5 @@ A typical :file:`Configuration/Services.yaml` may look like this:
    * TYPO3 uses the Symfony Dependency Injection component. The official documentation can be found at
      https://symfony.com/doc/current/service_container.html
 
-   * It is possible to use a glob pattern for the file paths, for example `'../Classes/*'`.
+   * A glob pattern is used to set the included files and their path, for example `'../Classes/**'`
```

Comment:
there should not be two stars

Reply:
See the documentation at https://en.wikipedia.org/wiki/Glob_(programming).

2 stars are needed to find all the files, and not only those from the subfolder but also from folders below the subfolders.

Reply:
Symony uses the own Finder component which does not follow natig glob() patterns. Don't mix things here.